### PR TITLE
Feature/asset posts crw 66

### PR DESCRIPTION
**Pull request by AbdallaEmadEldinAbdalla**
[CRW-66](https://techxarya.atlassian.net/browse/CRW-66)

- Implement the community widget.

[CRW-66]: https://techxarya.atlassian.net/browse/CRW-66?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/app/asset/assetInformation.tsx [hunk 1]
```diff
@@ -50,44 +53,61 @@ const AssetInformationTab: FC = () => {
...
+            preDefined
+            staticCoin={asset}
+          />}
+        </Col>
+      </Col>
+      <Col className="flex-[0.5] ps-6">
+        {/* Community widget */}
+        {posts.length > 0 && <>
+          <h3 className="font-extrabold text-white header-label pb-6">{t('communityTitle', { coin: asset.name })}</h3>
+          <Col className="w-full min-h-[300px] bg-black-2 rounded-md px-10 py-5">
```
Comment (almujtabamohammed) [inline src/components/app/asset/assetInformation.tsx:99]: background color isn't black-2

File: src/components/app/community/Post.tsx [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                </Col>
+            }
+
+            {
+                post?.FinancialData && <Row className='w-full min-h-[60px] bg-grey-3 rounded-md px-4 py-2 justify-between items-center my-2'>
+                    <Row className='gap-2 items-center'>
+                        <Image src={logoURL} alt='test' width={36} height={36} className='max-h-[36px]' />
+                        <Col className='gap-1'>
+                            <span className='text-white text-sm font-bold'>{post.FinancialData.Instrument?.MarketItem?.Symbol}/{post.FinancialData.Instrument?.MarketItem?.To}</span>
+                            <span className='text-white text-sm font-bold'>${post.FinancialData.PriceEvaluations?.[0].Price}</span>
```
Comment (almujtabamohammed) [inline src/components/app/community/Post.tsx:140]: Format number using formatNumber

File: src/components/app/community/Post.tsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+                        </Col>
+                    </Row>
+
+                    <Row className='items-center gap-4'>
+                        <Col className='gap-1'>
+                            {post?.FinancialData?.Sell != null && <Row className={clsx({ "bg-green-2": !post?.FinancialData?.Sell, "bg-red-2": post?.FinancialData?.Sell }, 'px-3.5 py-1 rounded-md items-center gap-1')}>
+                                {icon}
+                                <span className={clsx({ "text-green-1": !post?.FinancialData?.Sell, "text-red-1": post?.FinancialData?.Sell }, 'text-xs font-bold')}>{!post?.FinancialData?.Sell ? "Buy" : "Sell"}</span>
```
Comment (almujtabamohammed) [inline src/components/app/community/Post.tsx:148]: translate Buy and Sell

File: src/components/app/community/Post.tsx [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+                    <span className='text-xs font-bold text-blue-1'>{post.Category}</span>
+                </Col>
+                {post?.FinancialData?.Sell != null && <Row className={clsx({ "bg-green-2": !post?.FinancialData?.Sell, "bg-red-2": post?.FinancialData?.Sell }, 'px-3.5 py-1 rounded-md items-center gap-1')}>
+                    {icon}
+                    <span className={clsx({ "text-green-1": !post?.FinancialData?.Sell, "text-red-1": post?.FinancialData?.Sell }, 'text-xs font-bold')}>{!post?.FinancialData?.Sell ? "Buy" : "Sell"}</span>
+                </Row>}
+            </Row >
+            <p className='my-2 font-medium text-sm'>
+                {post.Text?.slice(0, post.Text.length > 140 ? 120 : post.Text.length)} {post?.Text?.length ? post?.Text?.length > 140 && "..." : ''}
+                {post?.Text?.length ? post?.Text?.length > 140 && <span className='text-blue-1 underline'> read more</span> : ''}
```
Comment (almujtabamohammed) [inline src/components/app/community/Post.tsx:104]: Translate read more

File: src/components/app/community/Post.tsx [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+                </Col>
+            </Row>
+
+            <Row className='w-[100%] items-center my-2 gap-2'>
+                <Col className='px-3.5 py-1 bg-blue-3 rounded-md'>
+                    <span className='text-xs font-bold text-blue-1'>{post.Category}</span>
+                </Col>
+                {post?.FinancialData?.Sell != null && <Row className={clsx({ "bg-green-2": !post?.FinancialData?.Sell, "bg-red-2": post?.FinancialData?.Sell }, 'px-3.5 py-1 rounded-md items-center gap-1')}>
+                    {icon}
+                    <span className={clsx({ "text-green-1": !post?.FinancialData?.Sell, "text-red-1": post?.FinancialData?.Sell }, 'text-xs font-bold')}>{!post?.FinancialData?.Sell ? "Buy" : "Sell"}</span>
```
Comment (almujtabamohammed) [inline src/components/app/community/Post.tsx:99]: Translate Buy and Sell

File: src/components/app/community/Post.tsx [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+                        <Col className='gap-1'>
+                            {post?.FinancialData?.Sell != null && <Row className={clsx({ "bg-green-2": !post?.FinancialData?.Sell, "bg-red-2": post?.FinancialData?.Sell }, 'px-3.5 py-1 rounded-md items-center gap-1')}>
+                                {icon}
+                                <span className={clsx({ "text-green-1": !post?.FinancialData?.Sell, "text-red-1": post?.FinancialData?.Sell }, 'text-xs font-bold')}>{!post?.FinancialData?.Sell ? "Buy" : "Sell"}</span>
+                            </Row>}
+                            {post.FinancialData.EntryValue && <span className='text-white text-sm font-bold'>Entry {post.FinancialData.EntryValue}</span>}
+                        </Col>
+                        <svg width="13" height="12" viewBox="0 0 13 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <path d="M5.79181 10.5719C6.18027 11.2189 7.1181 11.2189 7.50655 10.5719L12.4936 2.26471C12.8938 1.59819 12.4137 0.75 11.6363 0.75L1.6621 0.75C0.884694 0.75 0.404597 1.59819 0.804733 2.26471L5.79181 10.5719Z" fill="#ACB3BA" />
+                        </svg>
```
Comment (almujtabamohammed) [inline src/components/app/community/Post.tsx:154]: Use PlayIcon from heroicons

File: src/components/app/community/Post.tsx [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+                        </Col>
+                    </Row>
+
+                    <Row className='items-center gap-4'>
+                        <Col className='gap-1'>
+                            {post?.FinancialData?.Sell != null && <Row className={clsx({ "bg-green-2": !post?.FinancialData?.Sell, "bg-red-2": post?.FinancialData?.Sell }, 'px-3.5 py-1 rounded-md items-center gap-1')}>
+                                {icon}
+                                <span className={clsx({ "text-green-1": !post?.FinancialData?.Sell, "text-red-1": post?.FinancialData?.Sell }, 'text-xs font-bold')}>{!post?.FinancialData?.Sell ? "Buy" : "Sell"}</span>
+                            </Row>}
+                            {post.FinancialData.EntryValue && <span className='text-white text-sm font-bold'>Entry {post.FinancialData.EntryValue}</span>}
```
Comment (almujtabamohammed) [inline src/components/app/community/Post.tsx:150]: Translate Entry

File: src/components/app/community/AudioPlayer/AudioPlayer.tsx [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+                        <path d="M172.763 15.5386C172.391 15.5386 172.09 15.2397 172.09 14.8694V5.13038C172.09 4.76009 172.391 4.46118 172.763 4.46118C173.135 4.46118 173.435 4.76009 173.435 5.13038V14.8694C173.435 15.2397 173.135 15.5386 172.763 15.5386Z" fill="#ACB3BA" />
+                        <path d="M176.106 17.3144C175.734 17.3144 175.434 17.0155 175.434 16.6452V3.35499C175.434 2.9847 175.734 2.68579 176.106 2.68579C176.478 2.68579 176.779 2.9847 176.779 3.35499V16.6407C176.779 17.011 176.478 17.3099 176.106 17.3099V17.3144Z" fill="#ACB3BA" />
+                        <path d="M179.455 15.6146C179.083 15.6146 178.783 15.3157 178.783 14.9454V5.05469C178.783 4.68441 179.083 4.3855 179.455 4.3855C179.827 4.3855 180.127 4.68441 180.127 5.05469V14.9454C180.127 15.3157 179.827 15.6146 179.455 15.6146Z" fill="#ACB3BA" />
+                        <path d="M152.672 13.1698C152.3 13.1698 152 12.8709 152 12.5006V7.50391C152 7.13362 152.3 6.83472 152.672 6.83472C153.044 6.83472 153.345 7.13362 153.345 7.50391V12.5006C153.345 12.8709 153.044 13.1698 152.672 13.1698Z" fill="#ACB3BA" />
+                    </svg>
```
Comment (almujtabamohammed) [inline src/components/app/community/AudioPlayer/AudioPlayer.tsx:82]: Save image as asset and imported here

File: src/components/app/community/AudioPlayer/AudioPlayer.tsx [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+import Button from "../../../shared/buttons/button";
+import clsx from "clsx";
+
+export const playAudioIcon = (
+    <svg width="16" height="17" viewBox="0 0 16 17" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path
+            d="M0.166504 6.05512V10.9449C0.166504 13.8783 0.166504 15.345 1.12472 15.9297C2.08294 16.5144 3.38726 15.8436 5.99588 14.502L5.9959 14.502L10.7498 12.0572C13.7789 10.4993 15.2935 9.72042 15.2935 8.5C15.2935 7.27958 13.7789 6.50067 10.7498 4.94285L5.9959 2.49797C3.38726 1.15639 2.08295 0.485599 1.12472 1.07031C0.166504 1.65503 0.166504 3.12173 0.166504 6.05512Z"
+            fill="#01A385" />
+    </svg>
+)
```
Comment (almujtabamohammed) [inline src/components/app/community/AudioPlayer/AudioPlayer.tsx:13]: move svg to separate file

File: src/components/svg/arrowDown.tsx [hunk 10]
```diff
@@ -0,0 +1,6 @@
+export const ArrowDown = () => (
+    <svg width="9" height="9" viewBox="0 0 9 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M1 0.999959L7.50004 7.5" stroke="#B1454F" stroke-width="1.4" stroke-linecap="round" stroke-linejoin="round" />
+        <path d="M3.13037 7.5H7.50015V3.13022" stroke="#B1454F" stroke-width="1.4" stroke-linecap="round" stroke-linejoin="round" />
+    </svg>
+);
```
Comment (almujtabamohammed) [inline src/components/svg/arrowDown.tsx:6]: use PlayIcon from heroicons

File: src/components/svg/arrowUp.tsx [hunk 11]
```diff
@@ -0,0 +1,6 @@
+export const ArrowUp = () => (
+    <svg width="9" height="9" viewBox="0 0 9 9" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M1.38184 8.22227L8.14066 1" stroke="#22C55E" stroke-width="1.4" stroke-linecap="round" stroke-linejoin="round" />
+        <path d="M3.59766 1H8.1414V5.85531" stroke="#22C55E" stroke-width="1.4" stroke-linecap="round" stroke-linejoin="round" />
+    </svg>
+);
```
Comment (almujtabamohammed) [inline src/components/svg/arrowUp.tsx:6]: Use PlayIcon from heroicons

**Review by almujtabamohammed — CHANGES_REQUESTED** [review]
- Make assets page responsive
- Translate post component
- Match design colors
- Show loading while fetching posts, and add placeholder if there are no posts
- Image assets should be saved in /public/assets

File: src/components/app/asset/assetInformation.tsx (hunk 1 above)
Comment (AbdallaEmadEldinAbdalla) [reply src/components/app/asset/assetInformation.tsx:99]: The background color is not in our palette.

**Review by almujtabamohammed — APPROVED** [review]

**Pull request merged**